README.md

```diff
@@ -19,8 +19,8 @@ The following matrix shows the versions of Go and Cassandra that are tested with
 
 | Go/Cassandra | 4.0.x | 4.1.x | 
 |--------------|-------|-------|
-| 1.19         | yes   | yes   |
-| 1.20         | yes   | yes   |   
+| 1.22         | yes   | yes   |
+| 1.23         | yes   | yes   |
 
 Gocql has been tested in production against many versions of Cassandra. Due to limits in our CI setup we only
 test against the latest 2 GA releases.
```

Comment:
I debated about adding some text here about why we didn't include Cassandra 5.0.x here.  We probably need at least a GH issue around support for 5.0.x before we call it good... and then there's the need to get vector support in as well.  Anyways, in the end I decided not to include any additional text here but I'm very open to discussion on this point.

Reply:
As I see, Cassandra 5 is partially supported and works with the driver - https://github.com/apache/cassandra-gocql-driver/pull/1812. So, it can be added as a partially supported version. However, the vector type is not merged, and native protocol v4 should be used instead of v5 until it is merged.

Reply:
I believe that's correct @taaraora.  I fully expect the existing functionality of gocql will work pretty well with Cassandra 5.0.x.  What concerns me is [CASSANDRA-19906](https://issues.apache.org/jira/browse/CASSANDRA-19906).  We've been adding support for Cassandra 5.0.x to our test matrices for the other drivers DataStax maintains and we've hit an issue with precisely this problem on every one of them.  The PR you reference doesn't have this concern because it also moves to testcontainers, but that's a change (a) I'm not entirely sold on and (b) I certainly don't want to take on in order to get 1.7.0 out the door.

Given all of that I was fine leaving 5.0.0 out for now.  There's a [fix](https://github.com/apache/cassandra/commit/971747e3e25b7dec6a8ed50ed56ac0d14a3de6b1) to that issue coming in 5.0.1 so (barring some other major issue) I'd be fine with adding 5.0.1 to the existing setup once it's officially released (the vote is going on as I write this).

Make sense?